testsuite/setup.py

```diff
@@ -123,6 +123,8 @@
                   'data/dlpoly/CONFIG*',
                   'data/dlpoly/HISTORY*',
                   'data/*.xml',
+                  'coordinates/data/*xyz',
```

Comment:
Would it make more sense to keep all data files together?  If we keep going this way, then all the data files end up in one of ~5 directories, and because we reuse files between different tests (test_analysis will use a trj from test_coordinate), it will be annoying to remember where each file lives.